.devcontainer/Dockerfile

```
@@ -2,5 +2,14 @@ FROM ghcr.io/containerbase/node:18.16.0@sha256:fc48ac97eb5d0dee207d4a336a52d7d3b
 
 USER root
 
+RUN ls
+
+RUN apt update; \
+    apt install -y make g++
```

```suggestion
RUN install-apt make g++
```
use our helper

why do we need make and g++?

---

make and g++ are needed because re2 package is probably not available on ARM. Those tools are used for building the package while running yarn install.